### PR TITLE
section 08 - deploy to heroku with containers

### PR DESCRIPTION
**Pull request by ChristopherGS**
@solegalli this is the code for deploying a containerized app to Heroku - most of the files are the same as in previous sections, apart from those I have highlighted.

### CODE REVIEW

File: section-08-deploying-with-containers/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM python:3.9.4
```
Comment (ChristopherGS) [inline section-08-deploying-with-containers/Dockerfile:1]: Key change 1: Dockerfile updated for FastAPI (and python version updated)

File: section-08-deploying-with-containers/house-prices-api/run.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+uvicorn app.main:app --host 0.0.0.0 --port $PORT
```
Comment (ChristopherGS) [inline section-08-deploying-with-containers/house-prices-api/run.sh:1]: key change 2: Add bash script for docker deploy

File: .circleci/config.yml [hunk 3]
```diff
@@ -71,6 +71,27 @@ jobs:
           command: |
             tox -e publish_model
 
+
+  section_08_deploy_app_container_in_heroku:
```
Comment (ChristopherGS) [inline .circleci/config.yml:75]: Key change 0: Add new CI job

File: section-08-deploying-with-containers/Makefile [hunk 4]
```diff
@@ -0,0 +1,13 @@
+heroku-login:
```
Comment (ChristopherGS) [inline section-08-deploying-with-containers/Makefile:1]: Key change 4: Create section 08 Makefile with updated and correct Heroku docker commands. (Slightly simplified to not use the git tag in the docker image tag)

**Review by solegalli — APPROVED** [review]

**Pull request merged**